### PR TITLE
Splitting the landing_pages file into two

### PR DESCRIPTION
**Pull request by hannahbergam**
History: This test file has been through it. The most salient information: the test flakiness has quieted down since the fix to check for image.complete. Additionally, it's worth noting that the full test file runs in about 8 minutes, sometimes as high as 12. 

Unfortunately, image.complete is true for *any* of the following cases, source (https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/complete): 
Screenshot 2024-02-01 at 2 38 37 PM

This means that when an image errors or fails to load, the image will get marked complete and the test will move past that step and then fail. However, this is because the image has truly failed to load, which I believe is an issue with the images we should fix, not an issue with the test. The test *should* fail in these scenarios. 

What I propose is a simple solution that should help. By splitting the two most likely to fail (Dance and Oceans) across two files, if the tests continue to have some issues, the reruns are much more likely to pass. Additionally, if a DOTD should have to review them manually, the files will run much faster because they only contain half the cases. 

## Links

- spec: []()
- jira ticket: https://codedotorg.atlassian.net/browse/ACQ-1353

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by kelbyhawn — APPROVED** [review]
This looks good to me! I don't fully understand how tests work so you might want to wait for someone else to review before merging, but this makes sense based on your comment.

**Review by TurnerRiley — APPROVED** [review]

**Comment by TurnerRiley** [comment]
> Unfortunately, image.complete is true for _any_ of the following cases

Dang this makes it much more complicated :(

**Pull request merged**
